include/rocksdb/db.h

```diff
@@ -1410,9 +1410,8 @@ class DB {
 
 #endif  // ROCKSDB_LITE
 
-  // Sets the globally unique ID created at database creation time by invoking
-  // Env::GenerateUniqueId(), in identity. Returns Status::OK if identity could
-  // be set properly
+  // Reads the IDENTITY file and stores the id in the identity variable.
```

Comment:
I believe calling `GetIdentify(std::string&)` will not read from IDENTITY file. The db id is recovered during db recovery and stored in a variable `db_id_`.

Reply:
Checked the code and you are correct. It returns db_id_ which is set during opening. Should I change the comment of documentation reflecting the same?
It could says
"Returns the ID which is read from IDENTITY file during the opening of database by setting in the identity variable"

Reply:
Mostly LGTM. Maybe worthwhile to say "unique ID" as well.